pdns/ws-recursor.cc

```diff
@@ -620,50 +622,120 @@ void AsyncServer::newConnection()
 }
 
 // This is an entry point from FDM, so it needs to catch everything.
-void AsyncWebServer::serveConnection(std::shared_ptr<Socket> client) const
-try {
-  HttpRequest req;
-  YaHTTP::AsyncRequestLoader yarl;
-  yarl.initialize(&req);
-  client->setNonBlocking();
-
-  string data;
+void AsyncWebServer::serveConnection(std::shared_ptr<Socket> client) const {
+  const string logprefix = d_logprefix + "<" + to_string(getUniqueID()) + "> ";
```

Comment:
i'd favor removing the <> symbols, as I think they add little value, make parsing the message harder (would need stripping in awk etc) and sometimes can cause operational boo-boos (think inadvertent pasting of the log message into a shell instead of a textfile)